keps/0036-kudo-operator-toggle-behavior.md

```diff
@@ -0,0 +1,147 @@
+---
+kep-number: 36
+short-desc: KudoOperator Toggling
+title: Parameter-controlled toggling of KudoOperator tasks
+authors:
+  - "@asekretenko"
+owners:
+  - "@asekretenko"
```

Comment:
Historically owners have been committers and I don't know @asekretenko.  Are we expecting him to be a project committer?   There is usually a voting process by owners in the CNCF process for this.   That said there is a lot of flux in the committers and owners which is unfortunate.   

Please review the KEP process.  In particular:  https://github.com/kudobuilder/kudo/blame/main/keps/0001-kep-process.md#L168-L173 

```

The OWNER that is most closely associated with this KEP. If there is code or
 other artifacts that will result from this KEP, then it is expected that
 this OWNER will take responsibility for the bulk of those artifacts.
``` 
@asekretenko will you be able to serve in this capacity and own feature?

Reply:
@kensipe the intention is to have @asekretenko become an owner down the line and we're earning our way to it via contributions in the form of bug-fixes and other KEPs down the line.